android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -751,6 +751,9 @@ fun toPlayerViewConfig(json: ReadableMap) = PlayerViewConfig(
             ?.getBooleanOrNull("playbackSpeedSelectionEnabled")
             ?: true,
     ),
+    hideFirstFrame = json.getMap("hideFirstFrame")
```

Comment:
Should that first `getMap` be with `uiConfig`?

Note: I didn't run this yet but based on the code above that would make sense.

```suggestion
    hideFirstFrame = json.getMap("uiConfig")
```

Reply:
I think you're right that `json.getMap("hideFirstFrame")` is not correct, thanks for pointing out. However, changing it to your suggestion would mean that the `hideFirstFrame` flag is expected in `PlayerViewConfig.uiConfig` but it is in `PlayerViewConfig.hideFirstFrame`.

I think `hideFirstFrame = json.getBooleanOrNull("hideFirstFrame"),` would be the correct approach, but that doesn't work as I get a `Boolean?` but `PlayerViewConfig` expects a `Boolean`. Based on the conversation in the iOS part of this PR, we don't necessarily want to add the default value in RN, so I'm wondering what's the best approach for this. Maybe @matamegger has a suggestion?

Reply:
My bad for the wrong suggestion!
I see we already have some default values in there, which I missed. So no strong opinion about the default value topic.

Reply:
Makes all sense. I updated this in a108c8cd now.